code/game/machinery/computer/crew.dm

```diff
@@ -159,6 +167,8 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		JOB_ERT_CHAPLAIN = 225,
 		JOB_ERT_JANITOR = 226,
 		JOB_ERT_DEATHSQUAD = 227,
+		JOB_NT_REP = 230, // SKYRAT EDIT ADDITION
+		JOB_BLUESHIELD = 231, // SKYRAT EDIT ADDITION
```

Comment:
Is there a reason why not 228 and 229?

Reply:
Because basically, only roles ending with 0 will be bolded, and I wanted to make it clear that they're an authority figure, unlike the Blueshield.

Ideally I could make something where we could have them at the top of the list by the Captain, but, I didn't feel like making changes to literally the whole thing (although I could, it wouldn't be hard).

Reply:
Good enough for me